pilot/proxy/envoy/testdata/lds-v1-multi-match-fault.json.golden

```diff
@@ -637,16 +631,6 @@
           "quota_name": "RequestCount",
           "v2": {
            "defaultDestinationService": "hello.default.svc.cluster.local",
-           "forwardAttributes": {
-            "attributes": {
-             "source.ip": {
-              "bytesValue": "AAAAAAAAAAAAAP//CgEBAQ=="
```

Comment:
I saw those in the 0.4.0 headers, it's gone now ?

Reply:
Yes, I removed forwarded attributes from the list of attributes sent to the app. See https://github.com/istio/istio/pull/2419/files#diff-83699eda922e729afb2122093ca240b4R171